src/iconsManifest/supportedExtensions.ts

```diff
@@ -4513,15 +4513,29 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'vite',
-      extensions: ['vite.config.js', 'vite.config.ts'],
+      extensions: [
```

Comment:
It would be nice to change this similar to [this](https://github.com/vscode-icons/vscode-icons/blob/master/src/iconsManifest/supportedExtensions.ts#L528-L535)